README.adoc

```diff
@@ -1096,11 +1096,16 @@ as the code of the project relies on the
 https://github.com/fabric8io/kubernetes-client[Fabric8 Kubernetes Java client], which is a fluent DSL that can
 communicate by using `http` protocol to the REST API of the Kubernetes Server.
 
-To disable the integration with Kubernetes you can set `spring.cloud.kubernetes.enabled` to `false`. Please be aware that when `spring-cloud-kubernetes-config` is on the classpath,
```

Comment:
what we use to say : 

1a) if you wan to disable : use `spring.cloud.kubernetes.enabled=false`
2a) if you have config on the classpath, put `spring.cloud.kubernetes.enabled` into bootstrap
3a) if you want to disable `EnvironmentPostProcessor`, you need to do set `-DSPRING_CLOUD_KUBERNETES_ENABLED=false` 

What we say now:

1b) awareness is based on `ConditionalOnCloudPlatform`. It is possible to override
this via `spring.main.cloud-platform`. In this way we preserve (1a): You want to _disable_ integration? use `spring.main.cloud-platform`
2b) same as (1b) pretty much
3b) we do not mention  `-DSPRING_CLOUD_KUBERNETES_ENABLED=false`, because now, by default, it is _not_ enabled; and it was the case in (3a). As a matter of fact, the only reason we put this in the documentation was because it was enabled by default and users had to disable it. This paradigm shifts now in the reverse order. I mean how many people will have env variables `KUBERNETES_SERVICE_HOST` and `KUBERNETES_SERVICE_PORT` set? On the other hand, _everyone_ started with `spring.cloud.kubernetes.enabled=true`, by default, because of that `matchIfMissing=true`.